doc/source/changes/version_0_31.rst.inc

```diff
@@ -0,0 +1,21 @@
+﻿.. py:currentmodule:: larray_editor
+
+New features
+^^^^^^^^^^^^
+
+*
+
+
+.. _misc_editor:
+
+Miscellaneous improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*
+
+
+Fixes
+^^^^^
+
+* fixed exception raised when loading a session from a file
```

Comment:
it only failed if the session contained non-array objects, right?

Reply:
Yes, the changelog is indeed incomplete. Sorry.